git-versions.yml

```diff
@@ -50,7 +50,7 @@ services:
         image: mendersoftware/mender:3.5.1
 
     mender-connect:
-        image: mendersoftware/mender-connect:2.1.0
+        image: mendersoftware/mender-connect:2.1.1
```

Comment:
I cannot pull this image:
```bash
# docker pull mendersoftware/mender-connect:2.1.1
Error response from daemon: pull access denied for mendersoftware/mender-connect, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```
should it be like that?

Reply:
These are not Docker images (I know, I know).
It is just a way to express versions.